release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -516,6 +530,28 @@ Benefits of this feature include faster data processing, reduced CPU workloads,
 
 For more information, see xref:../networking/hardware_networks/configuring-hardware-offloading.adoc#about-hardware-offloading_hardware-offloading[Configuring hardware offloading].
 
+[id="ocp-4-10-openstack-kuryr-resource-reduction"]
+==== Reduction of {rh-openstack} resources created by Kuryr
+
+For clusters that run on {rh-openstack}, Kuryr now only creates Neutron networks and subnets for namespaces that have at least one pod on the pods network. Additionally, pools in a namespace are only populated after at least one pod on the pods network is created in the namespace.
+
+[id="ocp-4-10-openstack-dcn-for-enterprise"]
+==== Support for {rh-openstack} DCN (Technology Preview)
+
+You can now deploy a cluster on a {rh-openstack-first} deployment that uses a link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/distributed_compute_node_and_storage_deployment/index[distributed compute node (DCN)] configuration. There are a number of limitations:
+
+* Only {rh-openstack} version 16 is supported.
+* For {rh-openstack} 16.1.4, only hyper-converged infrastructure (HCI) and Ceph technologies are supported at the edge.
+* For {rh-openstack} 16.2, non-HCI and Ceph technologies are supported as well.
```

Comment:
Is your target audience going to understand the additional capabilities in 16.2?